graspy/inference/latent_position_test.py

```diff
@@ -64,22 +64,27 @@ class LatentPositionTest(BaseInference):
     n_bootstraps : int, optional (default 500)
         Number of bootstrap simulations to run to generate the null distribution
 
+    pass_graph : bool, optional (default True)
```

Comment:
try to keep line lengths on comments to <88 characters, black cannot fix comments